### DISCUSSION TITLE
too long building time

### DISCUSSION THREAD

**Discussion by snowden-fu** [ethereum/ethereum-org-website#11668] [Q&A] [answered]
I cloned repo to local computer, and it has compiled for over 5 min, is it normal time?

**Comment by corwintines** [comment, marked as answer]
Hey @fuzheng1998,

While the repo is large, have you setup your `GATSBY_BUILD_LANGS` in the .env file. I recommend using `GATSBY_BUILD_LANGS=en` to only build the site in english, or add the lang code you need as well. This will significantly reduce the build time for you locally.
